content/manager/update-deployment.md

```diff
@@ -6,58 +6,70 @@ draft: false
 weight: 650
 ---
 
-With Cloudify, you can update a deployment that was previously [created from a blueprint](http://docs.getcloudify.org/3.4.0/manager/create-deployment/). But what does 'updating' a deployment mean? Well, say you have a sizable intricate deployment of webservers and databases. After some time and research, you realize that you need to add a new kind of database, that should be connected to some of the existing webservers. 'Updating' a deployment means that instead of creating a new deployment from a blueprint that includes these new servers, you can simply add and connect these new databases to your existing deployment, while retaining the state of your current webservers-databases setting.
+With Cloudify, you can update a deployment that was previously [created from a blueprint](http://docs.getcloudify.org/3.4.0/manager/create-deployment/). But what does 'updating' a deployment mean? Well, say you have a sizable intricate deployment of webservers and databases. After some time and research, you realize that you need to add a new kind of database, that should be connected to some of the existing webservers. 'Updating' a deployment means that instead of *creating a new deployment* from a blueprint that includes these new servers, you can simply add and connect these new databases to your *existing deployment*, while retaining the state of your current webservers-databases setting.
```

Comment:
Currently, the sidebar only shows `h1` and `h2` headers. But in my opinion, the `h2` font size is way too big. So I'm only willing to change my `h3`s to `h2` when the font sizes will be revised.